include/cinder/TriMesh.h

```diff
@@ -200,7 +200,7 @@ class TriMesh : public geom::Source {
 	size_t		getNumIndices() const override { return mIndices.size(); }
 	//! Returns the total number of triangles contained by the TriMesh.
 	size_t		getNumTriangles() const { return mIndices.size() / 3; }
-	//! Returns the total number of indices contained by the TriMesh.
+	//! Returns the total number of verticies contained by the TriMesh.
```

Comment:
I was referring to these docs and this seemed like a bit of copy paste from a couple lines above.